scripts/aggregate.py

```diff
@@ -87,7 +88,12 @@ def failures_to_markdown(failures: Dict[str, List[str]], current_hash: str, patc
     with open("./labels.txt", "w") as f:
         f.write(f"{','.join(labels)}")
     result += "---\n\n"
-    result += failures_to_summary(failures)
+    summary, no_failures = failures_to_summary(failures)
+    result += summary
+    if no_failures:
+        # Something went wrong
+        labels.add("invalid")
```

Comment:
also this should be moved before the labels are added to result
```
if len(labels) > 0:
  result += f"labels: {', '.join(labels)}\n"
```

Reply:
Done